### PR TITLE
Step3task4

### PR DESCRIPTION
**Pull request by mikan373**
## What
- Deleted function related to json file
- Added items.db file
- Edited gitignore in order not to commit database
- Edited `POST /items` endpoint to save data in the database
- Edited `GET /items` endpoint to list all items in the database

## CHECKS :warning:

Please make sure you are aware of the following.
- [x] **Database is not commited
- [x] **The changes in this PR doesn't have private information

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -28,28 +30,31 @@ def root():
...
-    new_item = {"name": name, "category": category}
-    
-    with open(filename, "r") as file:
-        data = json.load(file)
-    
-    data["items"].append(new_item)
-
-    with open(filename, "w") as file:
-        json.dump(data, file)
+    dbname = "mercari.sqlite3"
```
Comment (tsoibet) [inline python/main.py]: dbname can be declared outside the function so every function can access to it.

File: python/main.py [hunk 2]
```diff
@@ -28,28 +30,31 @@ def root():
...
     return {"message": f"item received: {name}"}
 
+
 @app.get("/items")
 def show_item():
-    filename = "items.json"
 
-    with open(filename, "r") as file:
-        data = json.load(file)
+    dbname = "mercari.sqlite3"
```
Comment (tsoibet) [inline python/main.py]: here the dbname doesn't need to be declared again if you declared it outside.

**Review by tsoibet — APPROVED** [review]
LGTM

File: python/main.py (hunk 1 above)
Comment (mikan373) [reply python/main.py]: You're right, thanks for the comment!

**Pull request merged**